### ISSUE TITLE
Grid - className k-grid-edit-row is missing

### ISSUE THREAD

**Issue by dyabol** [telerik/kendo-react#600]
When i use grid with custom cell and editation, className **k-grid-edit-row** on row in edit is missing.

You can try it here:

https://stackblitz.com/edit/react-dhkpks-mq4gvc

When you uncomment columns without custom cell, **k-grid-edit-row** className is included.

**Comment by dyabol** [comment]
I tried fix it by **rowRender** function, but property **isInEdit** in **GridRowProps** is always false.

```
const rowRender = (trElement: React.ReactElement<HTMLTableRowElement>, props: GridRowProps) => {
    console.log(props.isInEdit);
    return <tr>{trElement.props.children}</tr>;
};
```

**Comment by nstoychev** [comment]
Fixed in dev channel - https://stackblitz.com/edit/react-dhkpks-mwqbxw.

**Comment by Xizario** [comment]
Available in 3.16.0
